packages/components/atoms/f-card/src/components/Card.vue

```diff
@@ -53,6 +60,10 @@ export default {
         isRounded: {
             type: Boolean,
             default: false
+        },
+        hasFullWidthBottomElement: {
```

Comment:
I'd be tempted to call this a footer, as that's what it is really?

So then you could call this `hasFullWidthFooter` and then the slot itself could just be called `cardFooter`